src/Pos/Context/Context.hs

```diff
@@ -75,15 +75,18 @@ import           Pos.Util.Util                 (postfixLFields)
 
 data LastKnownHeaderTag
 type LastKnownHeader ssc = TVar (Maybe (BlockHeader ssc))
-type MonadLastKnownHeader ssc ctx m = (MonadReader ctx m, HasLens LastKnownHeaderTag ctx (LastKnownHeader ssc))
+type MonadLastKnownHeader ssc ctx m
+     = (MonadReader ctx m, HasLens LastKnownHeaderTag ctx (LastKnownHeader ssc))
```

Comment:
Don't reformat this stuff without a purpose — you just plant merge conflicts for the future. If you want to fix it — create a `HasLastKnownHeader ssc ctx` class and remove the constraint synonym altogether.